app/src/main/res/xml/preferences.xml

```diff
@@ -166,6 +166,11 @@
                 android:summary="@string/pref_desc_ui_scrollOverBottom"
                 android:inputType="number"
                 android:defaultValue="3"/>
+            <CheckBoxPreference
+                android:key="@string/pref_key_ui_swipeArticles"
+                android:title="@string/pref_name_ui_swipeArticles"
+                android:summary="@string/pref_desc_ui_swipeArticles"
+                android:defaultValue="true"/>
```

Comment:
I would rather have it disabled by default. The feature is not implemented well and may be confusing. But it is debatable.